pf4j/pom.xml

```diff
@@ -33,6 +33,12 @@
             <version>1.7.5</version>
         </dependency>
         <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
Do we need this? In all my libraries projects I use slf4j-api as dependency and I use what slf4j I wish in application.